### PR TITLE
Feature/welcome page

### PR DESCRIPTION
**Pull request by partiallymisplaced**
Issue #4 
- [x] Placeholder messaging text 
- [x] Layout
- [x] Button to sign in with Github

03-screenshot-welcome

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,25 @@
 import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import logo from "./logo.svg";
-import Header from "./components/Header"
+import Header from "./components/Header";
 import Projects from "./pages/Projects";
+import Welcome from "./pages/Welcome";
 
 function App() {
   return (
```
Comment (nurmerey) [inline src/App.js]: Lets add a comment todo here to for fixing the route link

File: src/pages/Welcome.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+        <div className="platform-feature">
+          <h3>Title Explains Feature</h3>
+          <p>The feature summary paragraph will go into a bit more details. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque a mattis purus, vel cursus justo. Ut nec pretium ex.</p>
+        </div>
+      </section>
+      <section className="platform-featured-projects">
+        <h2>Featured Projects</h2>
+        <div className="featured-projects-block">
+          {/* <ProjectCard /> */}
```
Comment (nurmerey) [inline src/pages/Welcome.js]: Same, lets add a comment todo here to fix project card in this file with a new issue link

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
     <div className="App">
       <Header />
-      <Projects />
+      <Router>
```
Comment (partiallymisplaced) [inline src/App.js]: #12 Fix Routing

File: src/App.js (hunk 1 above)
Comment (partiallymisplaced) [reply src/App.js]: Done :)

File: src/pages/Welcome.js (hunk 2 above)
Comment (partiallymisplaced) [reply src/pages/Welcome.js]: Done :)

**Pull request merged**